bears/vcs/VCSCommitBear.py

```diff
@@ -0,0 +1,152 @@
+import abc
+import re
+
+from aenum import Flag
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.HiddenResult import HiddenResult
```

Comment:
Place this after the `coalib.misc.Shell` import. We usually also wanna sort the imports by the letter as well